layouts/_default/baseof.html

```diff
@@ -26,9 +26,30 @@
       <link rel="canonical" href="{{ .Permalink }}">
     {{ end }}
 
-    <link href="https://fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather:300,700%7CSource+Code+Pro:400,700&display=swap" rel="stylesheet">
```

Comment:
I'm not a CSS guru. Why can't the change just exclude remote assets by a parameter? Then the already included custom CSS option can embed the local fonts, CSS etc.?

Reply:
There is `use_default_remote_css` below. I'll change it to exclude only when a user wants custom ones.